cluster/images/hyperkube/master.json

```diff
@@ -37,6 +60,25 @@
               "--master=127.0.0.1:8080",
               "--v=2"
         ]
+    },
+    {
+      "name": "setup",
```

Comment:
@brendandburns How is it guaranteed that the `setup` container will create the files before `api-server` and `controller-manager` try to use them? Isn't there a race condition?